packages/components/src/font-size-picker/stories/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FontSizePicker from '../';
+
+export default { title: 'FontSizePicker', component: FontSizePicker };
+
+const FontSizePickerWithState = ( { ...props } ) => {
+	const [ fontSize, setFontSize ] = useState( 16 );
```

Comment:
Actually, you could use `knobs` with the `value` and the `fontSizes`.  This way folks could play with the default font size and all the possible font sizes provided. What do you think? @ItsJonQ will be able to help if that makes sense.

Reply:
I added the `fontSizes` to a knob, but the value I left as it was. The default `16` is only used on page load, after which point the state takes over. Since changing a knob doesn't reload the component in storybook, there wouldn't be any point in making the value a knob. Because changing it wouldn't reload the component, and reloading the page would reset it to the initial value. Unless I am mistaken about something? (Which is entirely possible 😄 )

Reply:
I gave it a spin and you are 100% correct. It isn't entirely what I assumed 🙂 

Let's leave it as is. I need to double-check other stories and update those which set the initial state with knobs as it doesn't reload as you pointed out.